src/index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+    data-qa="card"
+    class="card"
+    >
+      <div class="card__container">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+        <h1 class="card__item">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__code">
+          Product code: 195434
+        </p>
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="stars__rewiew">
+            Reviews: 5
+          </p>
+        </div>
+        <div class="card__price">
+          <p
+            class="card__price-item">
+            Price:
+          </p>
+          <p class="card__price-value">
+            $2,199
+          </p>
+        </div>
+        <a
+          href="#"
+          data-qa="hover"
+          class="card__button">
```

Comment:
```suggestion
          class="card__button"
        >
```
closing bracket should be on the same level as opening one